privaterelay/views.py

```diff
@@ -91,40 +90,40 @@ def profile_subdomain(request):
         return JsonResponse({"message": e.message, "subdomain": subdomain}, status=400)
 
 
-def send_ga_ping(ga_id: str, ga_uuid: str, data: Any) -> None:
-    try:
-        report(ga_id, ga_uuid, data)
-    except Exception as e:
-        logger.error("metrics_event", extra={"error": e})
-
-
 @csrf_exempt
 @require_http_methods(["POST"])
 def metrics_event(request: HttpRequest) -> JsonResponse:
+    """
+    Handle metrics events from the Relay extension.
+
+    This used to forward data to Google Analytics, but was not updated for GA4.
+
+    Now it logs the information and updates statsd counters.
+    """
     try:
         request_data = json.loads(request.body)
     except json.JSONDecodeError:
         return JsonResponse({"msg": "Could not decode JSON"}, status=415)
     if "ga_uuid" not in request_data:
         return JsonResponse({"msg": "No GA uuid found"}, status=404)
-    # "dimension5" is a Google Analytics-specific variable to track a custom dimension,
-    # used to determine which browser vendor the add-on is using: Firefox or Chrome
-    # "dimension7" is a Google Analytics-specific variable to track a custom dimension,
-    # used to determine where the ping is coming from: website (default), add-on or app
-    event_data = event(
-        request_data.get("category", None),
-        request_data.get("action", None),
-        request_data.get("label", None),
-        request_data.get("value", None),
-        dimension5=request_data.get("dimension5", None),
-        dimension7=request_data.get("dimension7", "website"),
-    )
-    t = threading.Thread(
-        target=send_ga_ping,
-        args=[settings.GOOGLE_ANALYTICS_ID, request_data.get("ga_uuid"), event_data],
-        daemon=True,
-    )
-    t.start()
+
+        # "real_address": sha256(new_emddail.encode("utf-8")).hexdigest(),
```

Comment:
question (non-blocking): where did this comment line come from? What's `new_emddail`?

Reply:
It's from here:

https://github.com/mozilla/fx-private-relay/blob/d878ad537cff5518036deaf61e5b9a0485471b76/privaterelay/views.py#L304

`emddail` is `email` and trying to run the vim delete line command `dd` when you are instead in insert mode.

I'll remove it.